package.json

```diff
@@ -219,6 +219,7 @@
 		"publish:check": "lerna updated",
 		"publish:dev": "npm run clean:package-types && npm run build:packages && lerna publish --dist-tag next",
 		"publish:legacy": "npm run clean:package-types && npm run build:packages && lerna publish --dist-tag legacy",
+		"publish:path": "npm run clean:package-types && npm run build:packages && lerna publish --dist-tag patch",
```

Comment:
We used publish:legacy in the past to achieve the same goal so we should remove it if we prefer patch. There is a typo in the name of the script 😃

Reply:
Bah! 

I kept looking at that wondering if that was the case! Thanks for looking, I'll update the PR.